object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/ObjectConstructionChecker.java

```diff
@@ -23,10 +27,21 @@
   "NoObligationStreams.astub",
   "IOUtils.astub"
 })
+@SupportedOptions({CHECK_MUST_CALL, COUNT_MUST_CALL})
 public class ObjectConstructionChecker extends CalledMethodsChecker {
 
   public static final String CHECK_MUST_CALL = "checkMustCall";
 
+  public static final String COUNT_MUST_CALL = "countMustCall";
+
+  /**
+   * The number of expressions with must-call obligations that were checked. Incremented only if the
+   * {@link #COUNT_MUST_CALL} option was supplied.
+   */
+  int numMustCall = 0;
+
+  int numMustCallFailed = 0;
```

Comment:
Do we really need to track this explicitly?  Is this number just equal to the number of errors reported?  If so, it may be more robust to count the # of errors with a script after the fact.  With this approach, we need to encapsulate all error reporting in a single routine, so we don't forget to increment the field.  But even with that, we need to remember to always call our own error-reporting routine, which is annoying.

@kelloggm thoughts?

Reply:
I agree with you. I think we should override `BaseTypeChecker#reportError` and increment this field there whenever we issue a `required.methods.not.called` error.

(I suggested this in another comment, which it seems I posted about 30 seconds later than you posted this!)

Reply:
Ok sounds good.  Let's go with overriding `BaseTypeChecker#reportError`

Reply:
https://github.com/kelloggm/object-construction-checker/pull/249/commits/5260a7c49c6dc2c042e59816683cc968c422883c